server/plugin/api.go

```diff
@@ -1285,17 +1286,17 @@ func (p *Plugin) getReposByOrg(c *UserContext, w http.ResponseWriter, r *http.Re
 	org := r.URL.Query().Get("organization")
 
 	if org == "" {
-		err := errors.New("organization query param is empty")
-		c.Log.WithError(err).Warnf("Failed to list repositories")
-		p.writeAPIError(w, &APIErrorResponse{Message: "Failed to fetch repositories", StatusCode: http.StatusInternalServerError})
+		err := errors.New("Organization query param is empty")
+		c.Log.WithError(err).Warnf("Organization query param is empty")
+		p.writeAPIError(w, &APIErrorResponse{Message: "Bad request, must include organization name ", StatusCode: http.StatusBadRequest})
```

Comment:
I don't think we need to write `Bad request` in the error, as it is already stated in the status code. Also, can we use the message as present in line 1289, as it tells us where we must include the organization name? What are your thoughts on this?

Reply:
I think you make a good point, we can just use the logging message as it's more descriptive. I've changed it include the message in line 1289.
```suggestion
		p.writeAPIError(w, &APIErrorResponse{Message: "Organization query is empty, must include organization name ", StatusCode: http.StatusBadRequest})
```